pandas-stubs/core/series.pyi

```diff
@@ -1024,8 +1024,8 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     def pop(self, item: Hashable) -> S1: ...
     def squeeze(self, axis: AxisIndex | None = ...) -> Scalar: ...
     def __abs__(self) -> Series[S1]: ...
-    def add_prefix(self, prefix: _str) -> Series[S1]: ...
-    def add_suffix(self, suffix: _str) -> Series[S1]: ...
+    def add_prefix(self, prefix: _str, axis: Axis | None = None) -> Series[S1]: ...
+    def add_suffix(self, suffix: _str, axis: Axis | None = None) -> Series[S1]: ...
```

Comment:
In `Series`, the only acceptable argument for `axis` is `AxisIndex | None`.  Also, remove the `= None` part and change it to `axis=...`

Reply:
done